src/main/java/com/customer/model/CreditCard.java

```
@@ -0,0 +1,55 @@
+package com.customer.model;
```

How does the response look like for request which does not have the format requested by `CreditCard` ?

How does the response look like in postman ?

---

You should throw a custom error (create a class of your own) instead of generic `javax` error

---

Done, Now it will throw a custom error message if any of the mandatory parameter is missing from the input request.